index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="kr">
```

Comment:
```suggestion
<html lang="ko">
```

kr이 아닌 ko가 올바른 문법입니다! :)